flytekit/bin/entrypoint.py

```diff
@@ -67,6 +67,137 @@ def _map_job_index_to_child_index(local_input_dir, datadir, index):
     return mapping_proto.literals[index].scalar.primitive.integer
 
 
+def _dispatch_execute(ctx: FlyteContext, task_def: PythonTask, inputs_path: str, output_prefix: str):
+    """
+    Dispatches execute to PythonTask
+        Step1: Download inputs and load into a literal map
+        Step2: Invoke task - dispatch_execute
+        Step3:
+            a: [Optional] Record outputs to output_prefix
+            b: OR if IgnoreOutputs is raised, then ignore uploading outputs
+            c: OR if an unhandled exception is retrieved - record it as an errors.pb
```

Comment:
this is not done yet right?  just making sure, i can do this next.

Reply:
ya not done yet, it raises the exception